.travis.yml

```diff
@@ -35,6 +35,20 @@ matrix:
       env:
         - PYTHON_VERSION=3.7
         - TOXENV=py37
+    - os: windows
+      language: shell
+      before_install: choco install python2
+      env:
+        - TOXENV=py27
+        - export PATH="/c/Python27:/c/Python27/Scripts:$PATH"
+    - os: windows
+      language: shell
+      before_install:
+        - curl https://www.python.org/ftp/python/3.7.0/python-3.7.0.exe -o /c/temp/python-3.7.0.exe
```

Comment:
Why not Python 3.7.5 directly? Or even 3.8.0?

Reply:
@BoboTiG I now experimented on ways to obtain Python environment in Travis. Choco's default python3 now Python 3.8. If it is ok to use last Python I've fix it now.